db/migrate/20191018212129_change_show_candidate_requirements_selection_column_default.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeShowCandidateRequirementsSelectionColumnDefault < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default :schools_school_profiles, :show_candidate_requirements_selection, true
```

Comment:
Is this needed, there are 2 migrations - the first changing the default, the second removing the column itself?